generators/entity-client/templates/react/src/main/webapp/app/entities/entity-update.tsx.ejs

```diff
@@ -435,7 +435,7 @@ _%>
                   <%= fieldNameHumanized %>
                 </Translate>
               </Label>
-              <AvField id="<%= entityFileName %>-<%= fieldName %>" type="text" name="<%= fieldName %>" <%- include react_validators %>/>
+              <AvInput id="<%= entityFileName %>-<%= fieldName %>" type="textarea" name="<%= fieldName %>" <%- include react_validators %>/>
```

Comment:
@agaspardcilia can you check if the validators applied still works for AvInput, some might be for AvField

Reply:
Are you sure those are still supported ? I mean when I try to create an entity with `jhe` it doesn't even ask me about any validation rule. I don't even know if it's normal. And if it's not supported anymore, why is it still available in the JDL. 
Am I missing something ?

Reply:
There should be validation for text blob field as per below
```
{
            when: response => response.fieldAdd === true && response.fieldValidate === true,
            type: 'checkbox',
            name: 'fieldValidateRules',
            message: 'Which validation rules do you want to add?',
            choices: (response) => {
                // Default rules applicable for fieldType 'LocalDate', 'Instant',
                // 'ZonedDateTime', 'UUID', 'Boolean', 'ByteBuffer' and 'Enum'
                const opts = [
                    {
                        name: 'Required',
                        value: 'required'
                    }/* ,
                    {
                        name: 'Unique',
                        value: 'unique'
                    } */
                ];
                if (response.fieldType === 'String' || response.fieldTypeBlobContent === 'text') {
                    opts.push(
                        {
                            name: 'Minimum length',
                            value: 'minlength'
                        },
                        {
                            name: 'Maximum length',
                            value: 'maxlength'
                        },
                        {
                            name: 'Regular expression pattern',
                            value: 'pattern'
                        }
                    );
                } else if (['Integer', 'Long', 'Float', 'Double', 'BigDecimal'].includes(response.fieldType)) {
                    opts.push(
                        {
                            name: 'Minimum',
                            value: 'min'
                        },
                        {
                            name: 'Maximum',
                            value: 'max'
                        }
                    );
                } else if (response.fieldType === 'byte[]' && response.fieldTypeBlobContent !== 'text') {
                    opts.push(
                        {
                            name: 'Minimum byte size',
                            value: 'minbytes'
                        },
                        {
                            name: 'Maximum byte size',
                            value: 'maxbytes'
                        }
                    );
                }
                return opts;
            },
```

Reply:
I guess I found a bug then.

Reply:
oh wait its disabled in the line above

```
        {
            when: response => response.fieldAdd === true && response.fieldType !== 'byte[]' && response.fieldType !== 'ByteBuffer',
            type: 'confirm',
            name: 'fieldValidate',
            message: 'Do you want to add validation rules to your field?',
            default: false
        },
```
I have no idea why its disabled @jhipster/developers does anyone remember why we removed validation for text fields

Reply:
Julien told me that it was because of the size limit. The server was downloading the complete file before making any check. It might be a good idea to let the user mark the field as `requiered` tho.

Reply:
yes, at least required should be there

Reply:
With validation enabled, everything works with text blobs but the `required`  validation is not working on regular blobs.